### PR TITLE
#131 Change text dunning document

### PR DESCRIPTION
**Pull request by cristinastefan90**
https://github.com/metasfresh/metasfresh-e2e/issues/131

### CODE REVIEW

File: cypress/integration/dunning/change_text_dunning_doc.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    });
+  });
+
+  it('Create bpartner', function() {
+    cy.fixture('sales/simple_customer.json').then(customerJson => {
+      const bpartner = new BPartner({ ...customerJson, name: businessPartnerName })
+        .setCustomer(true)
+        .setDunning(dunningTypeName)
+        .setPaymentTerm(paymentTerm)
+        .setBank(undefined);
```
Comment (TheBestPessimist) [inline cypress/integration/dunning/change_text_dunning_doc.js]: Typo here with `undefined` ?

File: cypress/integration/dunning/change_text_dunning_doc.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  it('Create sales invoice', function() {
+    cy.fixture('sales/sales_invoice.json').then(salesInvoiceJson => {
+      new SalesInvoice(businessPartnerName, salesInvoiceTargetDocumentType)
+        .addLine(new SalesInvoiceLine().setProduct(productName).setQuantity(originalQuantity))
+        .apply();
+    });
+    cy.completeDocument();
+  });
+  it('Sales Invoice is not paid', function() {
+    cy.getCheckboxValue('IsPaid').then(checkBoxValue => {
```
Comment (TheBestPessimist) [inline cypress/integration/dunning/change_text_dunning_doc.js]: There's also a cypress function: `cy.expectCheckboxValue('name', expectedValue)` which can be used here. (just fyi)

File: cypress/integration/dunning/change_text_dunning_doc.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      siRecordId = record;
+    });
+  });
+
+  it('Create Dunning Candidates', function() {
+    DunningCandidates.visit();
+    cy.executeHeaderActionWithDialog('C_Dunning_Candidate_Create');
+    cy.setCheckBoxValue('IsFullUpdate', true, true);
+    cy.pressStartButton();
+    // cy.wait(8000);
```
Comment (TheBestPessimist) [inline cypress/integration/dunning/change_text_dunning_doc.js]: cleanup: pls remove comment

File: cypress/integration/dunning/change_text_dunning_doc.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  });
+
+  it('Ensure there are exactly 2 Dunning Candidates - one for each dunning type level', function() {
+    DunningCandidates.visit();
+    filterBySalesInvoiceDocNumber(siDocumentNumber);
+    DunningCandidates.getRows().should('have.length', 2);
```
Comment (TheBestPessimist) [inline cypress/integration/dunning/change_text_dunning_doc.js]: alternative: `cy.expectNumberOfRows(2)` (just fyi)

File: cypress/support/commands/navigation.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import { RewriteURL } from "../utils/constants";
```
Comment (TheBestPessimist) [inline cypress/support/commands/navigation.js]: pls eslint this file. the imports should use `'` and not `"`

File: cypress/integration/dunning/change_text_dunning_doc.js (hunk 1 above)
Comment (cristinastefan90) [reply cypress/integration/dunning/change_text_dunning_doc.js]: removed.

File: cypress/integration/dunning/change_text_dunning_doc.js (hunk 2 above)
Comment (cristinastefan90) [reply cypress/integration/dunning/change_text_dunning_doc.js]: done.

File: cypress/integration/dunning/change_text_dunning_doc.js (hunk 3 above)
Comment (cristinastefan90) [reply cypress/integration/dunning/change_text_dunning_doc.js]: done

File: cypress/integration/dunning/change_text_dunning_doc.js (hunk 4 above)
Comment (cristinastefan90) [reply cypress/integration/dunning/change_text_dunning_doc.js]: you're right, done.

File: cypress/support/commands/navigation.js (hunk 5 above)
Comment (cristinastefan90) [reply cypress/support/commands/navigation.js]: done

**Pull request merged**